sorc/build_tropcy_NEMS.sh

```diff
@@ -24,13 +24,11 @@ fi
 module use ${cwd}/../modulefiles
 module load modulefile.storm_reloc_v6.0.0.$target
 
-if [ $target = "wcoss2" ]; then
-  SIGIO_LIB4=$SIGIO_LIB
-  SIGIO_INC4=$SIGIO_INC
-fi
+SIGIO_LIB4=$SIGIO_LIB
+SIGIO_INC4=$SIGIO_INC
```

Comment:
Why not just change the LIBS variable to use the original variable names?

Reply:
Done @ 40253b9. Note this build is waiting for the `libpng`/`png` module to be updated, see https://github.com/NOAA-EMC/hpc-stack/issues/379#issuecomment-1034187891